### PR TITLE
Roster - Enrollment Code Instructions Updated

### PR DESCRIPTION
**Pull request by darrenmason**
https://www.pivotaltracker.com/story/show/110106454

image

- [x] updated text
- [x] added book url link w/ ':3'

### CODE REVIEW

**Comment by darrenmason** [comment]
just noticed we said not to use <li>

**Comment by darrenmason** [comment]
i will update to check 'paste-ability'

**Comment by kathi-fletcher** [comment]
FYI -- I am going to see if I can find Dani/Courtney to check on the look. Maybe we do need another font after all.

**Comment by kathi-fletcher** [comment]
1. We meant for the new message to copy paste to students to be in addition to the original text that was there. Some of that instruction is still really important for teachers. 
2. We should have included "Ass you read ..." in the message to be copied, so that was our bad.
3. It is awfully long now, so Courtney is going to mess with it to see if she can make it clearer. 
4. In case the link code doesn't copy, we must include the actual link text at the end of the sentence which was the intention in the design. 
5. You already noted in the PR that the <li>'s  shouldn't be used so that the numbers also copy/paste correctly.
6. We are now thinking that some highlighting around the message to copy would be very desirable.
7. Courtney is condensing all the text (since we need the original stuff too which will make it even longer) and will ping us when she is done. It will be here: https://docs.google.com/document/d/1PxDDLzG0BlsAAXSCnZSgdY7kAgu7W9DWMb3fmzNQiwY/edit#heading=h.s2fw5o90az00

**Comment by kathi-fletcher** [comment]
Design is updated and condensed: Design is updated and condensed : https://docs.google.com/document/d/1PxDDLzG0BlsAAXSCnZSgdY7kAgu7W9DWMb3fmzNQiwY/edit#heading=h.s2fw5o90az00

Please adjust.

**Comment by darrenmason** [comment]
This is how it looks when copy/pasted from chrome into plain text:

As you read your OpenStax textbook online, you will come across embedded Concept Coach quizzes to help you understand and retain what you’ve read.

Follow the steps below to register for Concept Coach:

1. Paste this link in your web browser to visit your Biology textbook:
https://demo.cnx.org/contents/f10533ca-f803-490d-b935-88899941197f@3.1:3
2. Click on section 1.1 in the book. Scroll to the bottom of the section and click on the Concept Coach button.
3. Click "Sign up" and follow the prompts to create your free account.
4. At the end of your account set-up, you will be prompted to enter your two-word enrollment code: coach awesome
5. Continue to your Concept Coach questions!

File: src/components/course-settings/period-enrollment-code.cjsx [hunk 1]
```diff
@@ -11,20 +12,62 @@ module.exports = React.createClass
...
+        <p>
+          <span className='emphasis'>Copy the example message below </span>
+          to post in your Learning Management System or to email your students:
+        </p>
+      </div>
+
+      <div className='howto'>
+        <p>
+          As you read your OpenStax textbook online,
+           you will come across embedded Concept Coach quizzes to help you understand and retain what you’ve read.
```
Comment (philschatz) [inline src/components/course-settings/period-enrollment-code.cjsx]: can you change `quizzes` to `question sets`?

**Comment by kathi-fletcher** [comment]
Darren, sorry for all the wording changes. We found one more thing that needs addressing: 

Please change "quizzes" to "question sets" in this sentence 

As you read your OpenStax textbook online, you will come across embedded Concept Coach question sets to help you understand and retain what you’ve read.

File: src/components/course-settings/period-enrollment-code.cjsx (hunk 1 above)
Comment (darrenmason) [inline src/components/course-settings/period-enrollment-code.cjsx]: yup

**Comment by kathi-fletcher** [comment]
Looks good to me.

**Pull request merged**